web-app/src/components/BookingModal/BookingModalForm/container.js

```diff
@@ -67,7 +72,10 @@ const Container = Wrapped =>
         }
       } else if (name === 'isHalfday' && formData.isHalfday) {
         formData.end = formData.start;
-        formData.eventTypeId = 1;
+        const annualLeave = 1;
+        formData.eventTypeId = annualLeave;
```

Comment:
Would it not be better to use `formData.eventTypeId = eventTypes.ANNUAL_LEAVE` here?

Reply:
@William-Unosquare - Done. Also changed HOLIDAY to ANNUAL_LEAVE in eventTypes and updated files where neccessary. See updated commits below